Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/classes/Observations/ilObservationsCumulativeGUI.php

```diff
@@ -11,6 +11,7 @@ public function __construct($parent_obj) {
 
 		$this->gTpl = $tpl;
 		$this->parent_obj = $parent_obj;
+		$this->gTpl->addCSS("Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/templates/css/talent_assessment_observation_cummulative_table.css");
```

Comment:
Was ist in dieser Datei? Bitte nicht aus dem Skin laden sondern im Plugin hinterlegen. Sonst funktionieren die beiden nur zusammen.